js/forum/src/components/EmojiAreaButton.js

```diff
@@ -8,23 +8,18 @@ import icon from "flarum/helpers/icon";
 
 export default class EmojiAreaButton extends Component {
 
-    init() {
-        this.textEditor = null;
```

Comment:
Isn't that overkill ? The whole purpose of props is to be available to the class !? :neutral_face: